libtsuba/src/RDGTopology.cpp

```diff
@@ -483,8 +501,34 @@ tsuba::RDGTopology::DoStore(
     //TODO: emcginnis need different naming schemes for the optional topologies
     katana::Uri path_uri = MakeTopologyFileName(handle);
 
+    // file store must be bound to make a copy of it in a new location
+    if (!file_store_bound_) {
+      // Can't just use RDGTopology::Bind() here since path() is empty
+      // Must bind to old_path since we are persisting in a new location
+      std::string old_path = metadata_entry_->old_path_;
```

Comment:
I don't understand this comment. Why do we have to use old path?

Reply:
When we change the RDG location, we clear the PartitionTopologyMetadata path (and many other paths) to indicate we are moving the RDG.
Now since we likely don't have the topology bound at this point, we have to somehow find the topology file at the path we just cleared.
To make locating the topology file possible, when we clear the path we first save it in `old_path` so that we can bind and then store it here.
This code branch is only taken when we haven't modified the topology so we really just want to copy the topology file to the new RDG location.

Reply:
If that explanation makes sense, any suggestions on what to add to the comment to make things clearer?